### ISSUE TITLE
False positive FS003

### ISSUE THREAD

**Issue by MatthiasThul** [MichaelKim0407/flake8-use-fstring#6]
The below gives a false positive FS003 error.

```python
foo = re.compile(r'\(\w+Message\)":\s+({.*})}')
```
